c7n/resources/aws.py

```diff
@@ -381,6 +381,8 @@ def __init__(self, ctx, config=None):
         super(MetricsOutput, self).__init__(ctx, config)
         self.namespace = self.config.get('namespace', DEFAULT_NAMESPACE)
         self.region = self.config.get('region')
+        self.ignore_zero = self.config.get('ignore_zero')
+        self.metrics = self.config.get('metrics') and self.config.get('metrics').split(',')
```

Comment:
I'd prefer a more verbose variable name here, perhaps send_metrics or active_metrics .. 

also wondering if we should just default to the most common three ResourceCount, ApiCalls, and PolicyException. ActionTime, FilterTime I think are overly nuanced.

Reply:
I'd like the existing users won't be affected by the new query parameter. They should only need to be aware of it when they want to use it to filter metrics. So by default, if active_metrics is not set, everything remain the same. Does that make sense?

Reply:
yeah I get the compatibility but its not clear to me that the ActiveTime and FilterTime are useful at all, re sensible default. I'm okay deferring that post a community meeting discussion wrt.